hack/test-cmd.sh

```diff
@@ -78,9 +84,15 @@ set -e
 
 function find_tests {
   find "${OS_ROOT}/test/cmd" -name '*.sh' -not -wholename '*images_tests.sh' | grep -E "${1}" | sort -u
```

Comment:
If I returned a non-zero code from the function, then the script will terminate and run the `cleanup` function.  How would I propagate the `no_test_error` or something equivalent in that case?